File: VeamIt-project/Android/veam31000287/veam31000287/src/main/java/co/veam/veam31000287/ConsoleSellPdfPostHandler.java
package co.veam.veam31000287;

import java.util.ArrayList;

/**
 * Created by veam on 11/8/16.
 */
public class ConsoleSellPdfPostHandler implements HandlePostResultInterface {
    SellPdfObject sellPdf ;
    PdfObject pdf ;

    public SellPdfObject getSellPdf() {
        return sellPdf;
    }

    public void setSellPdf(SellPdfObject sellPdf) {
        this.sellPdf = sellPdf;
    }

    public PdfObject getPdf() {
        return pdf;
    }

    public void setPdf(PdfObject pdf) {
        this.pdf = pdf;
    }

    @Override
    public void handlePostResult(ArrayList<String> results) {
        //NSLog(@"%@::handlePostResult",NSStringFromClass(this.class])) ;
        if(results.size()  >= 2){
            //NSLog(@"count >= 2") ;
            String result = results.get(0) ;
            if(result.equals("OK")){
                String sellPdfId = results.get(1) ;
                String pdfId = results.get(2) ;

                sellPdf.setId(sellPdfId) ;
                pdf.setId(pdfId) ;
            }
        }
    }
}

File: VeamIt-project/iOS/veam00000000/ConsoleSellPdfPostHandler.m
//
//  ConsoleSellPdfPostHandler.m
//  veam00000000
//
//  Created by veam on 11/5/15.
//  Copyright (c) 2015 veam. All rights reserved.
//

#import "ConsoleSellPdfPostHandler.h"
#import "ConsoleUtil.h"

@implementation ConsoleSellPdfPostHandler

@synthesize sellPdf ;
@synthesize pdf ;

- (void)handlePostResult:(NSArray *)results
{
    //NSLog(@"%@::handlePostResult",NSStringFromClass([self class])) ;
    if([results count] >= 2){
        //NSLog(@"count >= 2") ;
        NSString *result = [results objectAtIndex:0] ;
        if([result isEqualToString:@"OK"]){
            NSString *sellPdfId = [results objectAtIndex:1] ;
            NSString *pdfId = [results objectAtIndex:2] ;
            
            [sellPdf setSellPdfId:sellPdfId] ;
            [pdf setPdfId:pdfId] ;
        }
    }
}

@end
